tests/platform/cloud/AcceptanceTests_oauth2_privilegedPrefsCreator.txt

```diff
@@ -0,0 +1,7 @@
+AcceptanceTests_privilegedPrefsCreator.js
+
+Description:
+This is a series of tests using the online flowmanager to test the workflow of that privileged preferences creators to add new preferences sets using OAuth2 client credentials grant. 
```

Comment:
I think we can remove the "that" in "test the workflow of that privileged preferences creators".